assets/src/dashboard/components/cardGallery/index.js

```diff
@@ -109,6 +109,10 @@ function CardGallery({ children }) {
     };
   }, [updateContainerSize]);
 
+  useEffect(() => {
+    setActiveCardIndex(0);
+  }, [children]);
```

Comment:
This makes sure the active card in the gallery gets reset when we toggle through templates.